Makefile

```diff
@@ -281,6 +282,11 @@ $(ISO_ROOT)/images/$(NGINX_IMAGE_NAME)-$(NGINX_IMAGE_VERSION).tar.gz: \
 	IMAGE=$(NGINX_IMAGE)
 $(ISO_ROOT)/images/$(NGINX_IMAGE_NAME)-$(NGINX_IMAGE_VERSION).tar.gz: \
 	IMAGE_TAG=$(NGINX_IMAGE_NAME):$(NGINX_IMAGE_VERSION)
+$(ISO_ROOT)/images/$(SALT_MASTER_IMAGE_NAME)-$(SALT_MASTER_IMAGE_VERSION).tar.gz:
+	mkdir -p $(dir $@)
+	$(eval IMAGE_TAG=$(SALT_MASTER_IMAGE_NAME):$(SALT_MASTER_IMAGE_VERSION) )
```

Comment:
There's no real reason to `eval` here. Just use the relevant values in-place below.